lookups/lookup-get-cname-example-1/lookup-get-cname-example-1.5.x.cs

```diff
@@ -0,0 +1,24 @@
+// Download the twilio-csharp library from twilio.com/docs/libraries/csharp
+using System;
+using System.Collections.Generic;
+using Twilio;
+using Twilio.Rest.Lookups.V1;
+using Twilio.Types;
+
+public class Example
+{
+	public static void Main(string[] args)
+	{
+		// Find your Account SID and Auth Token at twilio.com/console
+		const string accountSid = "ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX";
+		const string authToken = "your_auth_token";
+
+		TwilioClient.Init(accountSid, authToken);
+
+		var phoneNumber = PhoneNumberResource.Fetch(
+		    new PhoneNumber("+15108675309"),
+		    type: new List<string> { "caller-name" });
+
```

Comment:
That kind of sanity check is outside of the scope of any snippet. We want to focus in the **happy path** most of the time.